### ISSUE TITLE
Great Work!

### ISSUE THREAD

**Issue by KrishaYue** [pursehouse/kinect-activity-reaction-interface#1]
Is this will work on kinect v1? This is really awesome !!!

**Comment by pursehouse** [comment]
yeah I built it for v1. I started working on a v2 upgrade but didn't have time to finish it

**Comment by pursehouse** [comment]
you might have to roll it back to before the v2 commits

**Comment by KrishaYue** [comment]
I tried to run this in VS 2015  but It didnt work. 

I have kinect v1 plugged in

**Comment by KrishaYue** [comment]
help me to run this please :'(

**Comment by pursehouse** [comment]
yeah you'll have to roll back to the commits before I started changing it to v2. "it didn't work" isn't any info for me to help from
